packages/credentials/tests/verifiable-credential.spec.ts

```diff
@@ -409,4 +410,35 @@ describe('Verifiable Credential Tests', () => {
       sinon.restore();
     });
   });
+
+  describe('Web5TestVectorsCredentials', () => {
+    it('verify', async () => {
+      const vectors = CredentialsVerifyTestVector.vectors;
+
+      for (const vector of vectors) {
+        const { input, errors, description } = vector;
+
+        // TODO: DID:JWK is not supported yet
+        if (description === 'verify a jwt verifiable credential signed with a did:jwk') {
```

Comment:
@nitro-neal can you add an issue in the `web5-js` repo to ensure we double back and remove this conditional skip once a `did:jwk` is merged for `@web5/dids`?

Reply:
added - https://github.com/TBD54566975/web5-js/issues/371